vmicore/src/os/PageProtection.h

```diff
@@ -1,10 +1,10 @@
 #ifndef VMICORE_PAGEPROTECTION_H
 #define VMICORE_PAGEPROTECTION_H
 
-#include "OperatingSystem.h"
 #include <cstdint>
 #include <string>
 #include <vmicore/os/IPageProtection.h>
+#include <vmicore/os/OperatingSystem.h>
```

Comment:
Uhm, you just deleted this file. Are you sure this is correct?

Reply:
Yes

Reply:
Ah, I see it now. All good.